core/scripts/keystone/src/05_deploy_initialize_capabilities_registry.go

```diff
@@ -93,20 +93,24 @@ var (
 	}
 	targetDonPeers = []peer{
 		{
-			PeerID: "12D3KooWJrthXtnPHw7xyHFAxo6NxifYTvc8igKYaA6wRRRqtsMb",
-			Signer: "0x3F82750353Ea7a051ec9bA011BC628284f9a5327",
+			PeerID: "12D3KooWNBr1AD3vD3dzSLgg1tK56qyJoenDx7EYNnZpbr1g4jD6",
```

Comment:
We should still keep the old entries since they're used for EVM and add a `aptosTargetDonPeers` instead?

Reply:
Yes we should, unless the intention is not to merge this PR and just to run it against the existing CapabilitiesRegistry. @smickovskid can you confirm?

Reply:
Yeah, this is a temporary PR since I don't think we should start adding chains here. This will be done via G++ in the future

Reply:
Since this is used to manage staging for the time being I'd prefer we add the Aptos nodes here and get the PR merged